TinCan/RemoteLRS.cs

```diff
@@ -97,21 +111,13 @@ private MyHTTPResponse MakeSyncRequest(MyHTTPRequest req)
                 url += req.resource;
             }
 
-            if (req.queryParams != null)
+            String qs = "";
+            qs = AppendParamsToExistingQueryString(qs, req.queryParams);
+            qs = AppendParamsToExistingQueryString(qs, extended);
```

Comment:
This isn't necessarily wrong, but is a bit different behavior than how the original instance of this works. Specifically, in TinCanJS [here](https://github.com/RusticiSoftware/TinCanJS/blob/master/src/LRS.js#L276), we only include the value from `extended` if it isn't in the passed in props. We should be cognizant of the difference and whether it is intentional. It seems the other libs don't yet have support for `extended` either, so what we do here sets a bit of a precedent.

Reply:
That was not intentional. I will add the same logic that TinCanJS has.